lib/model/txproposal.js

```diff
@@ -171,8 +182,7 @@ TxProposal.prototype.getBitcoreTx = function() {
 };
 
 TxProposal.prototype.getNetworkName = function() {
-  var someAddress = this.toAddress || this.outputs[0].toAddress;
-  return Bitcore.Address(someAddress).toObject().network;
+  return Bitcore.Address(this.changeAddress.address).toObject().network;
```

Comment:
This topic [was already discussed in BWU](https://github.com/bitpay/bitcore-wallet-utils/pull/27#discussion-diff-36182237). I think, if BWU assumes all txps to have changeAddress (see link above), we may safely assume the same in BWS. What do you think?